developers/buildsystem.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: developersguide
+title: Buildsystem
+---
+
+{% include base.html %}
+
+# Build system
```

Comment:
```suggestion
# Build System
```
We normally use capitals in headers throughout the documentation. Could you please go through all newly added sections and adapt them accordingly?

Reply:
If you find time, can you identify those and propose more commitable suggestions? I see this as a joint effort, it cannot be done by a single person. I will also have a look when time permits of course.

Reply:
Sure, just wanted to be sure that you do any further headers already with capitals.